### ISSUE TITLE
NotImplementedError on initial stem training

### ISSUE THREAD

**Issue by mabjw** [autonomousvision/stylegan_xl#16]
Hello,
coming from Projected GAN I wanted to test Stylegan-XL. Upon installing/updating all dependencies and wanting to train a model from scratch on my windows PC, I am getting the following Error with the following command:
python train.py --outdir=training-runs --cfg=stylegan3-t --data=T:\Set_sg_xl_v1.zip --gpus=1 --mirror=1 --batch=64 --snap=4 --stem --syn_layers=10

Last output of cmd shell:
```
Constructing networks...
loaded imagenet embeddings from in_embeddings/tf_efficientnet_lite0.pkl: Embedding(1000, 320)
Traceback (most recent call last):
  File "I:\Github\stylegan_xl\train.py", line 362, in <module>
    main()  # pylint: disable=no-value-for-parameter
  File "C:\Users\[username]\AppData\Local\Programs\Python\Python39\lib\site-packages\click\core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "C:\Users\[username]\AppData\Local\Programs\Python\Python39\lib\site-packages\click\core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "C:\Users\[username]\AppData\Local\Programs\Python\Python39\lib\site-packages\click\core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "C:\Users\[username]\AppData\Local\Programs\Python\Python39\lib\site-packages\click\core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "I:\Github\stylegan_xl\train.py", line 347, in main
    launch_training(c=c, desc=desc, outdir=opts.outdir, dry_run=opts.dry_run)
  File "I:\Github\stylegan_xl\train.py", line 104, in launch_training
    subprocess_fn(rank=0, c=c, temp_dir=temp_dir)
  File "I:\Github\stylegan_xl\train.py", line 49, in subprocess_fn
    training_loop.training_loop(rank=rank, **c)
  File "I:\Github\stylegan_xl\training\training_loop.py", line 171, in training_loop
    D = dnnlib.util.construct_class_by_name(**D_kwargs, **common_kwargs).train().requires_grad_(False).to(device) # subclass of torch.nn.Module
  File "I:\Github\stylegan_xl\dnnlib\util.py", line 303, in construct_class_by_name
    return call_func_by_name(*args, func_name=class_name, **kwargs)
  File "I:\Github\stylegan_xl\dnnlib\util.py", line 298, in call_func_by_name
    return func_obj(*args, **kwargs)
  File "I:\Github\stylegan_xl\pg_modules\discriminator.py", line 172, in __init__
    feat = F_RandomProj(bb_name, **backbone_kwargs)
  File "I:\Github\stylegan_xl\pg_modules\projector.py", line 109, in __init__
    self.normstats = get_backbone_normstats(backbone)
  File "I:\Github\stylegan_xl\pg_modules\projector.py", line 33, in get_backbone_normstats
    raise NotImplementedError
```
    
All dependencies as listed in the environment.yml are installed.
Additionally:
- Python 3.9.10
- PyTorch 1.10.2+cu113 with matching CUDA Toolkit (Was working 100% fine with projected_gan!)
- Compiler is the previously working setup from ProjectedGAN with Visual Studio compilers

My extremely limited python understanding tells me that somehow the `backbone` variable in projector.py line 109 does not get populated somehow?

**Comment by xl-sr** [comment]
do you have the current version? it is likely fixed by [ec0f21a](https://github.com/autonomousvision/stylegan_xl/commit/ec0f21a6ec5b16d8a16f8e23147498defa303cdc)

**Comment by mabjw** [comment]
> 

In fact no!
I updated my local repo and things are working now!
Thanks so much for the hint ;)
